docs/modules/ROOT/pages/how-tos/exoscale/change_storage_node_size.adoc

```diff
@@ -0,0 +1,494 @@
+  -p '[{
+    "op": "replace",
+    "path": "/spec/healthCheck/daemonHealth/mon",
+    "value": {
+      "disabled": false,
+      "interval": "10s",
+      "timeout": "10s"
+    }
+  }]'
+kubectl --as=cluster-admin -n syn-rook-ceph-operator delete pods \
+  -l app=rook-ceph-operator
+----
+
+. Wait for operator to settle.
+Wait for a log message saying `done reconciling ceph cluster in namespace "syn-rook-ceph-cluster"`
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-operator logs -f \
+  deploy/rook-ceph-operator
+----
+
+. Cordon old nodes
++
+[source,bash]
+----
+for node in $(echo -n $NODES_TO_REPLACE); do
+  kubectl --as=cluster-admin cordon "${node}"
+done
+----
++
+. For every id in `$MON_IDS` replace the MON pod
++
+[source,bash]
+----
+mon_id=<MON_ID>
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster delete pod \
+  -l app=rook-ceph-mon,ceph_daemon_id="${mon_id}"
+
+# Wait until new MON is scheduled
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster get pods -w
+
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster exec -it deploy/rook-ceph-tools -- \
+  ceph status
+
+# Repeat for all other $MON_IDS
+----
+
+. Verify that three MONs are running
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster get deploy -l app=rook-ceph-mon
+----
+
+. Reset the MON failover timeout
++
+[NOTE]
+====
+We currently have to restart the operator to force it to pick up the new MON health check configuration.
+Once https://github.com/rook/rook/issues/8363[Rook.io GitHub issue #8363] is fixed, the operator restart shouldn't be necessary anymore.
+====
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster patch cephcluster cluster --type=json \
+  -p '[{
+    "op": "replace",
+    "path": "/spec/healthCheck/daemonHealth/mon",
+    "value": {}
+  }]'
+kubectl --as=cluster-admin -n syn-rook-ceph-operator delete pods \
+  -l app=rook-ceph-operator
+----
+
+=== Clean up the old nodes
+
+. Drain the nodes to replace
++
+[source,bash]
+----
+for node in $(echo -n $NODES_TO_REPLACE); do
+  kubectl --as=cluster-admin drain "${node}" \
+    --delete-emptydir-data --ignore-daemonsets
+done
+----
+
+. Delete the nodes from the cluster
++
+[source,bash]
+----
+for node in $(echo -n $NODES_TO_REPLACE); do
+  kubectl --as=cluster-admin delete node "${node}"
+done
+----
+
+. Remove the Exoscale VMs
++
+[source,bash]
+----
+for node in $(echo -n $NODES_TO_REPLACE); do
+  node_id=$(exo vm list -O json | \
+    jq --arg storage_node "$node" -r \
+    '.[] | select(.name==$storage_node) | .id')
+
+  echo "Removing node:"
+  exo vm list | grep "${node_id}"
+  echo -n "Is this correct? (y/N) "; read REPLY; echo
+  if [[ $REPLY =~ ^[Yy]$ ]]; then
+    exo vm delete "${node_id}"
+  fi
+done
+----
+
+. Clean up localstorage PV of decommissioned node
++
+[source,bash]
+----
+kubectl --as=cluster-admin delete pv "${pv_name}"
+----
```

Comment:
There is more than just one PV. You probably meant to iterate over `old_pv_names`